sia/miner_test.go

```diff
@@ -19,3 +19,12 @@ func mineSingleBlock(t *testing.T, c *Core) {
 		t.Error(err)
 	}
 }
+
+func testMinerDeadlocking(t *testing.T, c *Core) {
+	threads := 1
+	c.MinerInfo()
+	c.UpdateMiner(threads)
+	c.StartMining()
+	c.MinerInfo()
+	c.StopMining()
+}
```

Comment:
this test seems superfluous if the locking issue has been fixed. It looks more like debugging code than testing code.